docs/src/types/B512.md

```diff
@@ -0,0 +1,13 @@
+# B512
+
+In the Rust SDK, the `B512` definition matches the Sway standard library type with the same name and will be converted accordingly when interacting with contracts:
```

Comment:
Didn't we say we would avoid saying `Sway` in the docs? I never fully understood why, @digorithm it feels like saying `Fuel` here wouldn't make sense either. Wdyt?